docs/ref/class-based-views/generic-editing.txt

```diff
@@ -216,7 +216,8 @@ editing content:
     * :class:`django.views.generic.base.TemplateResponseMixin`
     * ``django.views.generic.edit.BaseDeleteView``
     * :class:`django.views.generic.edit.DeletionMixin`
-    * ``django.views.generic.detail.BaseDetailView``
+    * :class:`django.views.generic.edit.FormMixin`
+    * :class:`django.views.generic.edit.ProcessFormView`
```

Comment:
We need a `versionchanged` here, and then some discussion of the `form_class` attribute. (It defaults to a no-op form, but if you adjust that can require, e.g. a check-box.)

Reply:
Added `versionchanged`.